src/sagemaker_tensorflow/pipemode.py

```diff
@@ -53,6 +53,8 @@ def __init__(self, channel, record_format='RecordIO',
             state_dir: The directory where pipe index state is persisted.
             config_dir: The path for SageMaker input data config.
             benchmark: If True, causes the Dataset to emit timing and throughput metrics to stdout.
+            benchmark_records_interval: The number of records per interval to emit timing and throughput metrics
+                                        to stdout. If zero, no emitting the metrics.
```

Comment:
This documentation is a little confusing - to the reader, it's not clear how ``benchmark`` and ``benchmark_records_interval`` interrelate.

benchmark: Controls whether to emit timing and throughput metrics after closing an Iterator created from this Dataset. If True, timing and throughput metrics will be emitted to stdout after an Iterator created from this Dataset is destroyed.
benchmark_records_interval: Controls whether to emit timing and throughput metrics while records are being read from this Dataset. Defines the number of records per interval to emit timing and throughput metrics. If zero, no metrics will be emitted while records are being read from this Dataset. Metrics are emitted to stdout.

Reply:
fixed in the new revision.